Automattic/sensei
# Add Featured Video Block

### Changes proposed in this Pull Request

* Adds a Featured Video Block to the Lesson post type.

### Testing instructions

1. Check this PR out
2. Go to a Lesson and add a `Featured Video` block
3. Try adding a video from your Media Library
4. Try adding an embed from YouTube/VideoPress/Vimeo
5. Save the Lesson, make sure the frontend looks the same

## Known Potential Issues
Becuase we don't limit what Embeds they can use, it's possible but unlikely that a user could switch the embed to something that is NOT a video.
The user would need to manually remove the Video block that's in the template and then manually add another Embed that is not a video. 

### Screenshot / Video
![featured-video-block2](https://user-images.githubusercontent.com/3220162/189724658-0590f95a-8248-4a6c-8d48-b73b734f0970.gif)
